aten/src/ATen/native/cuda/BatchLinearAlgebraLib.cpp

```diff
@@ -490,6 +669,8 @@ std::tuple<Tensor, Tensor, Tensor> _svd_helper_cuda_lib(const Tensor& self, bool
     VT_working_copy.transpose_(-2, -1);
   }
 
+  // VT_working_copy is row-major, actually "V" now
+
   // heuristic for using `gesvdjBatched` over `gesvdj`
   if (m <= 32 && n <= 32 && batch_size > 1 && (!some || m == n)) {
     apply_svd_lib_gesvdjBatched(self, U_working_copy, S_working_copy, VT_working_copy, infos, compute_uv);
```

Comment:
Since we don't have a switch to switch the backend of `gesvd`, `gesvdj`, or `gesvdjBatched`, I can't find a good way to test this `gesvd` implementation in the CI. 

I tested this locally by replacing the `apply_svd_lib_gesvdj` below with `apply_svd_lib_gesvd`, and all svd tests passed.